script.js

```diff
@@ -7,10 +6,39 @@ function createProductImageElement(imageSource) {
   return img;
 }
 
+function cartItemClickListener(event) {
+
+}
+
+function createCartItemElement({ sku, name, salePrice }) {
+  const li = document.createElement('li');
+  li.className = 'cart__item';
+  li.innerText = `SKU: ${sku} | NAME: ${name} | PRICE: $${salePrice}`;
+  li.addEventListener('click', cartItemClickListener);
+  return li;
+}
+
+function apiItem(event) {
+  const itemId = event.path[1].firstChild.innerText;
+  fetch(`https://api.mercadolibre.com/items/${itemId}`)
+    .then(response => response.json())
+    .then((data) => {
+      const cartItem = {
+        sku: data.id,
+        name: data.title,
+        salePrice: data.price,
+      };
+      document.querySelector('ol').appendChild(createCartItemElement(cartItem));
```

Comment:
```suggestion
document.querySelector('.items').appendChild(createCartItemElement(element));
```

Tem o destructuring na chegada da funcao que você pode renomear

line 13
function createCartItemElement({ id: sku, title: name, salePrice }) {